src/nodes/QskBoxRendererEllipse.cpp

```diff
@@ -532,8 +546,29 @@ namespace
         {
         }
 
+        void addAdditionalLines( float x11, float y11, float x12, float y12, // start line
+                                 float x21,float y21, float x22, float y22, // end line
+                                 const QskGradient& gradient, Line* lines )
+        {
+            int additionalStopCount = additionalGradientStops( gradient );
+
+            auto s = gradient.stops();
+
+            for( int i = 1; i <= additionalStopCount; ++i )
+            {
+                auto p = ( 1 - s.at( i ).position() );
+                float xStart = x11 + p * ( x21 - x11 ),
+                    yStart = y11 + p * ( y21 - y11 ),
+                    xEnd = x12 + p * ( x22 - x12 ),
+                    yEnd = y12 + p * ( y22 - y12 );
+
+                lines[ additionalStopCount - i + 1 ].setLine( xStart, yStart,
+                    xEnd, yEnd, s.at( i ).color() );
+            }
+        }
+
         template< class BorderMap, class FillMap >
-        inline void createLines( Qt::Orientation orientation, Line* borderLines,
+        void createLines( Qt::Orientation orientation, Line* borderLines,
```

Comment:
Why removing the inline keyword ?

Reply:
Qt has this rule of thumb where you should only inline methods that have ~10 lines or so, otherwise the code gets too big. Since we don't use this function that often, I have reintroduced the inline keyword again.